src/utils/streams.js

```diff
@@ -0,0 +1,142 @@
+import program from 'commander';
+import fs from 'fs';
+import * as path from 'path';
+import readline from 'readline';
+import through2 from 'through2';
+import concat from 'concat';
+import { jsonTransform } from './convertor';
+
+function reverse(str) {
+  return str
+    .split('')
+    .reverse()
+    .join('');
+}
+
+function transform() {
+  process.stdout.write('Enter a string: ');
+  process.stdin
```

Comment:
this can be done w/o `through2` usage, just with 
```
process.stdin.on('data', data => { // your code for transforming })
```

Reply:
@AlexBryuzgin but in the task it is mentioned to use `through2`